### DISCUSSION TITLE
Veloce, few questions about correspondence

### DISCUSSION THREAD

**Discussion by ijm8710** [lichess-org/mobile#999] [General]
Hi Veloce, saw you just pushed PR #997. Thanks.

There were a few other similar incidents (and a couple enhancements) relating to correspondence that I had seen and tried asking about. 

I figured maybe moving to a discussion post may be more helpful. Any chance to please provide commentary on if you think any should also be resolved or if you would like me to provide more info on them.

- Recieve alert when opponent plays their move in a correspondence game. But this time, instead of opening the alert, instead simply open the app. The feed should be automatically registering that it is now your turn, but it is failing to do so until the app is manually refreshed 
- open alert (the game will actually register it is your turn this time), but instead of playing a move, backswipe back to home-feed and you will see it think that it’s not your turn.
  - the other bug here on this one is this double opening of the game. When I find the game is lagging in terms of being at the wrong place, I backswipe back. After the first backswipe I’m brought back to another version of the exact same game in the exact same place. Eventually I return back to homefeed where everything can eventually update and sync to the proper spot
- still wish there was easier way to see which games are your turn (grey out ones that aren’t, show all games that are your turn without having to click "see all games")
- correspondence games on home feed should show how much time opponent has left before its game over (their perspective, not yours when it's their turn)
- once you play your move in your correspondence game it should automatically jump to your next available game or jump back to home feed if none are left
- on desktop, for the activity feed on your profile, correspondence games actually link to the game itself. App should do similarly

**Comment by veloce** [comment]
Hi! First 2 points are now resolved in the challenges branch I'm working on.

**Reply by ijm8710** [reply]
Thanks Veloce, would you say any of items 3-5 make sense when you have time? 6 I will admit is purely just a request but I think 3-5 are pretty necessary to clearly communicate the game screen. 

And lastly, any chance to take a look at issue #812 when you get a chance. It's not correspondence related, but it's another example of things not syncing correctly. Not requesting you to prioritize it any differently but just wanted to confirm you can reproduce it and agree it should be looked into when you do in fact have time 😀 

Thanks!
